cmd2/argcomplete_bridge.py

```diff
@@ -228,15 +228,14 @@ def __call__(self, argument_parser, completer=None, always_complete_options=True
                 output_stream.write(ifs.join(completions).encode(argcomplete.sys_encoding))
             elif outstr:
                 # if there are no completions, but we got something from stdout, try to print help
-
                 # trick the bash completion into thinking there are 2 completions that are unlikely
                 # to ever match.
-                outstr = outstr.replace('\n', ' ').replace('\t', ' ').replace('    ', ' ').strip()
-                # generate a filler entry that should always sort first
-                filler = ' {0:><{width}}'.format('', width=len(outstr)/2)
-                outstr = ifs.join([filler, outstr])
 
-                output_stream.write(outstr.encode(argcomplete.sys_encoding))
+                comp_type = int(os.environ["COMP_TYPE"])
```

Comment:
Strange.  But if it works, cool.

Reply:
Yeah. The first tab press comes in as 9. Makes total sense.